services/app-web/src/constants/routes.ts

```diff
@@ -29,23 +30,25 @@ const STATE_SUBMISSION_FORM_ROUTES = [
     'SUBMISSIONS_CONTACTS',
     'SUBMISSIONS_DOCUMENTS',
     'SUBMISSIONS_REVIEW_SUBMIT',
+    'SUBMISSIONS_SUMMARY',
 ] as RouteT[]
 
 const RoutesRecord: Record<RouteT, string> = {
     ROOT: '/',
     AUTH: '/auth',
     DASHBOARD: '/dashboard',
     HELP: '/help',
-    SUBMISSIONS: '/submissions',
-    SUBMISSIONS_NEW: '/submissions/new',
-    SUBMISSIONS_FORM: '/submissions/:id',
-    SUBMISSIONS_TYPE: '/submissions/:id/type',
-    SUBMISSIONS_CONTRACT_DETAILS: '/submissions/:id/contract-details',
-    SUBMISSIONS_RATE_DETAILS: '/submissions/:id/rate-details',
-    SUBMISSIONS_CONTACTS: '/submissions/:id/contacts',
-    SUBMISSIONS_DOCUMENTS: '/submissions/:id/documents',
-    SUBMISSIONS_REVIEW_SUBMIT: '/submissions/:id/review-and-submit',
-    SUBMISSIONS_REVISION: '/submissions/:id/revisions/:revisionVersion',
+    SUBMISSIONS: 'submissions',
+    SUBMISSIONS_NEW: 'submissions/new',
+    SUBMISSIONS_FORM: 'submissions/:id/form/*',
+    SUBMISSIONS_TYPE: 'type',
+    SUBMISSIONS_CONTRACT_DETAILS: 'contract-details',
+    SUBMISSIONS_RATE_DETAILS: 'rate-details',
+    SUBMISSIONS_CONTACTS: 'contacts',
+    SUBMISSIONS_DOCUMENTS: 'documents',
+    SUBMISSIONS_REVIEW_SUBMIT: 'review-and-submit',
+    SUBMISSIONS_SUMMARY: 'submissions/:id/summary',
+    SUBMISSIONS_REVISION: 'submissions/:id/revisions/:revisionVersion',
```

Comment:
So `/submissions/:id/summary/revisions/:revisionVersion` is also a route now? That's what I see farther down in the tests. I'm gonna check this out 🔍

Reply:
`/submissions/:id/summary/revisions/:revisionVersion` I think I was thinking about nesting revisions in summary, but reverted back. This route in the test mention above was missed when reverting.